wicket-util/pom.xml

```diff
@@ -60,6 +59,12 @@
 						<goals>
 							<goal>test-jar</goal>
 						</goals>
+						<configuration>
+							<includes>
+	          					<include>org/apache/wicket/util/license/**</include>
+	          					<include>org/apache/wicket/util/markup/xhtml/**</include>
+	        				</includes>						
+						</configuration>
```

Comment:
👍🏻

Reply:
You are welcome :) On the contrary, writing fixed class paths as text in code or xml files are often sources of errors if people forget to adapt, e. g. when moving packages.